src/ansible_navigator/_yaml.py

```diff
@@ -22,14 +22,20 @@
 
 
 class YamlStyle(NamedTuple):
-    """the params for yaml dump"""
+    """The parameters for yaml dump."""
 
     default_flow_style: bool = False
     explicit_start: bool = True
 
 
 def human_dump(obj: Any, filename: str = None, fmode: str = "w") -> Union[str, None]:
-    """Consistant dumping of object across the application"""
+    """Consistant dumping of object across the application.
```

Comment:
1) There's a typo in "Consistent"
2) Functions/methods represent actions and their docstrings should start with a verb. Does it do something or is it just a static concept you're describing?

```suggestion
    """Dump an object consistently across the application.
```

(you could optionally expand on what it actually means in a separate paragraph)

Reply:
got it